maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -101,6 +101,13 @@ private void checkVersionsEqual(String v1, String v2) {
         assertEquals(c2, c1, "expected " + v2 + ".equals( " + v1 + " )");
     }
 
+    private void checkVersionsEqualOrder(String v1, String v2) {
+        Comparable c1 = newComparable(v1);
+        Comparable c2 = newComparable(v2);
```

Comment:
Inlining newComparable is probably also a good idea. Reading it now, I don't know what this method does.